command/issue.go

```diff
@@ -352,11 +352,11 @@ func issueCreate(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	var templateFiles []string
+	var nonLegacyTemplateFiles []string
 	if baseOverride == "" {
```

Comment:
Not exactly related to this PR, but why are we ignoring the value of the `repo` flag? :thinking: